### PR TITLE
Updating mysqli: mysqli_close

### PR DESCRIPTION
**Pull request by kamil-tekiela**
This page was missing its own example. I have also replaced the old shared note, with a better description. The link was not particularly useful.

### CODE REVIEW

File: reference/mysqli/mysqli/close.xml [hunk 1]
```diff
@@ -22,7 +22,14 @@
...
    Closes a previously opened database connection.
   </para>
-  &mysql.close.connections.result.sets;
+  <para>
+   Open non-persistent MySQL connections and result sets are automatically
+   destroyed when a PHP script finishes its execution. Explicitly closing open 
+   connections and freeing result sets is optional. However, it's a good idea
+   to close the connection as soon as the script finishes performing all of its
+   database operations, if it still has a lot of processing to do after
+   getting the results.
```
Comment (cmb69) [inline reference/mysqli/mysqli/close.xml:31]: I think we should not talk about finishing a script, but rather about the objects being destroyed.

Comment (kamil-tekiela) [reply reference/mysqli/mysqli/close.xml:31]: Ok, can you suggest how to word it better?

Comment (kamil-tekiela) [reply reference/mysqli/mysqli/close.xml:31]: How about this?

>Open non-persistent MySQL connections and result sets are automatically destroyed along with their objects.

Comment (cmb69) [reply reference/mysqli/mysqli/close.xml:31]: Yes, that's good. Or maybe explicitly refer to "close", e.g. "Open non-persistent MySQL connections and result sets are automatically closed when their objects are destroyed."

**Comment by cmb69** [comment]
Frankly, this looks like suggesting general optimizations, which apply to a lot of objects (e.g. `GdImage`s), but that appears to be quite fitting to ext/mysqli. :)

Thank you for the PR!

**Pull request merged**